Ginger/GingerCoreNET/Run/RunListenerLib/CenteralizedExecutionLogger/AccountReportExecutionLogger.cs

```diff
@@ -71,14 +71,23 @@ public async Task SendRunsetExecutionDataToCentralDbTaskAsync(RunSetConfig runse
 
         public async Task RunSetEnd(RunSetConfig runsetConfig)
         {
+
             await SendRunsetEndDataToCentralDbTaskAsync(runsetConfig);
+
             Reporter.HideStatusMessage();
         }
 
         public async Task SendRunsetEndDataToCentralDbTaskAsync(RunSetConfig runsetConfig)
         {
             AccountReportRunSet accountReportRunSet = AccountReportEntitiesDataMapping.MapRunsetEndData(runsetConfig, mContext);
-            await AccountReportApiHandler.SendRunsetExecutionDataToCentralDBAsync(accountReportRunSet, true);
+            bool isDataUploadedOnCentralDb = await AccountReportApiHandler.SendRunsetExecutionDataToCentralDBAsync(accountReportRunSet, true);
+
+            LiteDBRepository liteDB = new();
```

Comment:
Will it work with text file logger ?

Reply:
This function is not invoked when Text File is selected in the UI.